docs/source/install/prerequisites.rst

```diff
@@ -205,30 +209,73 @@ Make sure that RabbitMQ is running with:
     Remove the mentioned option from the file ``/usr/libexec/rabbitmq/rabbitmq-env`` and restart the server.
     If you still have trouble getting RabbitMQ to run, please refer to the detailed instructions provided on the `website of RabbitMQ itself for generic Unix systems <https://www.rabbitmq.com/install-generic-unix.html>`_.
 
+
 .. _details_wsl:
 
 Windows Subsystem for Linux (Ubuntu)
 ====================================
 
-The guide for Ubuntu above can be followed but there are a few things to note:
-
-#. The `windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
-   should be installed and started,
-   since the version in WSL Ubuntu 18.04 does not work properly.
+The guide for Ubuntu above can generally be followed, but there are a few things to note:
 
 #. Linux services under WSL are not started automatically.
-   To start the PostgreSQL service, type the command below in the terminal::
+   To start the PostgreSQL and RabbitMQ-server services, type the command below in the terminal::
 
      sudo service postgresql start
+     sudo service rabbitmq-server start
+
+   .. tip::
+
+       These services may be run at startup *without* passing a password in the following way:
+
+       Create a ``.sh`` file with the lines above, but *without* ``sudo``.
+       Make the file executeable, i.e., type::
+
+         chmod +x /path/to/file.sh
+
+       Then type::
+
+         sudo visudo
+
+       And add the line::
+
+         <username> ALL=(root) NOPASSWD: /path/to/file.sh
 
-#. There is a `known issue <https://github.com/Microsoft/WSL/issues/856>`_ in WSL Ubuntu 18.04 where the timezone is not
-   configured correctly out-of-the-box, which may cause problem for the database.
+       Replacing ``<username>`` with your Ubuntu username.
+       This will make you able to run *only* this specific ``.sh`` file with ``root`` access, i.e., not passing a password, without lowering security on your system.
+
+#. There is a `known issue <https://github.com/Microsoft/WSL/issues/856>`_ in WSL Ubuntu 18.04 where the timezone is not configured correctly out-of-the-box, which may cause problem for the database.
    The following command can be used to re-configure the time zone::
 
      sudo dpkg-reconfigure tzdata
 
 #. The file open limit may need to be raised using ``ulimit -n 2048`` (default is 1024), when running tests.
    You can check the limit by using ``ulimit -n``.
 
+   .. hint:: This may need to be run every time the system starts up.
+
 It may be worth considering adding some of these commands to your ``~/.bashrc`` file, since some of these settings may reset upon reboot.
 
+.. hint::
+
+    The Windows Subsystem for Linux (WSL) 2 is recommended.
+    It is available from Windows 10 builds 18917 or higher.
+    For more information, see `the WSL documentation <https://docs.microsoft.com/en-us/windows/wsl/wsl2-install>`_.
+
```

Comment:
Maybe I am a bit conservative with this, but I feel that we should not advertise the user to try WSL 2 immediately since the feature is not fully launched on stable windows builds yet. Things may not work out of the box, for example, accessing Window/Linux services using `localhost` (e.g. open Jupyter Notebook served inside). The gain of having a real Linux kernel and faster IO may not have huge impact on daily use. 
We should clearly state that the instructions apply to WSL 1 or WSL 2 in the documentation though.

Reply:
This is completely fair. I will state however that running `jupyter notebook --no-browser` in WSL Ubuntu works perfectly and `localhost:8888` is accessible using WSL1 on Windows 10 version 1909 build 18363.476 :+1: